apps/common/controllers/postcode-loop.js

```diff
@@ -70,33 +54,35 @@ module.exports = class PostcodeLoopController extends BaseController {
   }
 
   removeItem(req, res) {
-    const items = req.sessionModel.get(`${this.field}Addresses`);
-    req.sessionModel.set(`${this.field}Addresses`, _.omit(items, req.params.id));
-    const step = _.size(items) > 1 ? `/${this.field}-add-another-address` : `/${this.field}-postcode`;
+    const items = req.sessionModel.get(`${this.options.locals.field}Addresses`);
```

Comment:
this removing address functionality could be abstracted to the baseAddress controller as its used by every page